src/DataFrames.jl

```diff
@@ -9,16 +9,23 @@ module DataFrames
 
 using Reexport, StatsBase, SortingAlgorithms, Compat
 @reexport using CategoricalArrays, Missings
-using Base: Sort, Order
+using Base.Sort
+using Base.Order
 using Compat.Unicode
 using Compat.Printf
+using Compat: @warn
 
 if VERSION >= v"0.7.0-DEV.2738"
+    using Unicode: normalize
     const kwpairs = pairs
 else
     kwpairs(x::AbstractArray) = (first(v) => last(v) for v in x)
+    using Compat.IOBuffer
+    import Base.LinAlg: normalize
```

Comment:
There was a Compat update so I was really hoping I could do this properly, but it's still really screwed up.  When you do `using Compat.Unicode: normalize` in 0.6 it doesn't import the `normalize_str` methods.  Anyway, this can't be removed, because it throws an error telling you that you must explicitly import `LinAlg.normalize`.  Sorry I've been hesitant to make PR's to Compat, but there are so many gotchas and existing bugs that doing that is rather difficult.

This should *not* break if Compat gets fixed, especially because `Unicode` doesn't even export `normalize`.

Reply:
OK. Doesn't it work if you do `using Compat.Unicode` and then `Unicode.normalize` for every use?

Reply:
BTW, you should be able to use it soon (don't forget to update REQUIRE): https://github.com/JuliaLang/METADATA.jl/pull/13615

Reply:
awesome, thanks, I will wait until METADATA gets updated and try it out.